nab/detectors/bayes_changept/README.md

```diff
@@ -0,0 +1,5 @@
+# Bayesian Online Changepoint Detection
```

Comment:
These comments are helpful. Let's put them in the class docstring rather than a separate README file, similar to what we've done for the Gaussian detector.